configuration/README.md

```diff
@@ -92,6 +92,27 @@ Since no receiver matcher was provided, it will match any method beginning with
 
 As just two examples, this may be used to avoid analyzing test code, or to suppress "false positive" reports.
 
+### Suppressing false positives
+
+The analyzer may produce reports on pieces of code that are actually safe. If you run across such a "false positive" report, you may suppress it by adding a comment to the line where the taint reached the `sink`:
+
+```
+// levee.DoNotReport
+mysinks.SinkF("here's a value: %v", safeValue)
+```
+
+You may include a justification in the suppression comment, e.g.:
+
+```
+// levee.DoNotReport: I have verified that "safeValue" cannot actually contain sensitive data (mlevesquedion, Feb 05 2021).
+```
+
+In fact, as long as the exact string `levee.DoNotReport` appears at the beginning of a line in a comment above or directly on the line where a report would be produced, the report will be suppressed.
```

Comment:
Elaborate about not handling suppression comments in the middle of multi-line calls.  See test comment.

Reply:
Done.